src/index.ts

```diff
@@ -4,11 +4,12 @@ import prepareNGINX from "./nginx";
 import shell from "./utils/shell";
 import fs from "fs";
 import { config } from "./config";
+import axios from "axios";
+
 async function main() {
-  if (!process.env._DAPPNODE_GLOBAL_DOMAIN) {
-    console.error("DAppManager did not inject enviornment. Quitting.");
-    process.exit(1);
-  }
+  const dappnodeDomain = process.env._DAPPNODE_GLOBAL_DOMAIN
```

Comment:
I would add more verbosity weather or not the env is empty

Reply:
Done. For API retrieving there is already verbose logging